pkg/simulator/terraform.go

```diff
@@ -20,12 +20,12 @@ func PrepareTfArgs(cmd string, bucket string) []string {
 
 	if cmd == "init" || cmd == "plan" || cmd == "apply" || cmd == "destroy" {
 		arguments = append(arguments, "-input=false")
-		arguments = append(arguments, "--var-file=settings/bastion.tfvars")
+		arguments = append(arguments, "--var-file="+s.TfVarsDir+"/settings/bastion.tfvars")
```

Comment:
Prefer `fmt.Sprintf()` over string concatenation.  It's consistent with lower down - we should try to do that everywhere.